src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -498,8 +498,19 @@ def load_command_table(self, _):
         g.custom_command('stop', 'stop_webapp')
         g.custom_command('start', 'start_webapp')
         g.custom_command('restart', 'restart_webapp')
+        g.generic_update_command('update', getter_name="get_functionapp", setter_name='set_functionapp', exception_handler=update_function_ex_handler_factory(),
+                                 custom_func_name='update_functionapp', getter_type=appservice_custom, setter_type=appservice_custom, command_type=webapp_sdk)
 
     with self.command_group('logicapp', custom_command_type=logicapp_custom) as g:
         g.custom_command('create', 'create_logicapp', exception_handler=ex_handler_factory())
         g.custom_command('list', 'list_logicapp', table_transformer=transform_web_list_output)
         g.custom_show_command('show', 'show_logicapp', table_transformer=transform_web_output)
+        g.custom_command('scale', 'scale_logicapp', exception_handler=ex_handler_factory())
+
+    with self.command_group('logicapp config appsettings', custom_command_type=logicapp_custom) as g:
+        g.custom_command('list', 'get_logicapp_app_settings', exception_handler=empty_on_404)
+        g.custom_command('set', 'update_logicapp_app_settings', exception_handler=ex_handler_factory())
+        g.custom_command('delete', 'delete_logicapp_app_settings', exception_handler=ex_handler_factory())
+
+    with self.command_group('logicapp deployment source') as g:
+        g.custom_command('config-zip', 'enable_zip_deploy_functionapp')
```

Comment:
using functionapp logic does the exceptional handling to check if SCM can be reached included? @StrawnSC ? were you changes for webapp only?

Reply:
@haroonf what is the update on this one?

Reply:
@haroonf I added this try/except block to the logic app zip deploy code: https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L671
Maybe something similar for logic apps is needed

Reply:
Ah okay, looks like I misremembered. Thanks for checking @haroonf

Reply:
@StrawnSC AFAIK this is call where we call publishing profile API to call SCM? if so, remember if a user enables BasicAuth the publishing profile call will fail but shouldn't stop them from making calls to SCM directly for Zip deploy or other such calls.